spec/turtle.bnf

```diff
@@ -23,10 +23,10 @@ String                ::= STRING_LITERAL_QUOTE | STRING_LITERAL_SINGLE_QUOTE
 iri                   ::= IRIREF | PrefixedName
 PrefixedName          ::= PNAME_LN | PNAME_NS
 BlankNode             ::= BLANK_NODE_LABEL | ANON
-quotedTriple          ::= '<<' qtSubject predicate qtObject '>>'
-qtSubject             ::=	iri | BlankNode | quotedTriple
-qtObject              ::=	iri | BlankNode | literal | quotedTriple
-annotation            ::= '{|' predicateObjectList '|}'
+reifier               ::= '<<' ((iri | BlankNode) '|' )? subject predicate object '>>'
```

Comment:
In https://github.com/w3c/rdf-concepts/pull/96 the term `reifier` refers to the identifier part of this construct, not the whole construct itself. Alternatives might be:

* `indirectTripleTerm`
* `reifiedTripleTerm`
* `tripleReference`

Reply:
I think `reifiedTripleTerm` is precise?

I thought about `tripleDescriptor` too. Ping @afs &mdash; I think you initially had that as a provisional name for what we now call triple terms. Various definitions exist (["serves to describe or identify"](https://www.merriam-webster.com/dictionary/descriptor), [data descriptor](https://en.wikipedia.org/wiki/Data_descriptor)).  Might be more obscure. Then again, "annotation" isn't clear either, without definition, that it means "a reifier of this asserted triple"; but I still think "annotation" is a good name for the syntax.

*If* `<< ... >>` grows to allow multiple triples inside (["triple blocks"](https://github.com/w3c/rdf-star-wg/issues/116#issuecomment-2234357279)), the name should still work ("simple" vs. "compound"?).

(I agree that these should also work standalone (["reification declaration"](https://github.com/w3c/rdf-star-wg/issues/116#issuecomment-2234357279));  i.e. having the `predicateObjectList` optional.)